plugins/imjournal/imjournal.c

```diff
@@ -776,6 +784,7 @@ CODESTARTbeginCnfLoad
 	cs.iDfltFacility = DFLT_FACILITY;
 	cs.bUseJnlPID = -1;
 	cs.usePid = NULL;
+	cs.bWorkAroundJournalBug = 0;
```

Comment:
@richm , @rgerhards added the explicit initialization here. If there is some new config structure I could do that as a separate PR after this is merged...